woocommerce/woocommerce
# Monorepo CLI tooling: Set folder organisation

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Add new folder structure for Monorepo CLI tools. Each section will have a folder under `/src` and be able to create its own command and any number of subcommands.

```
pnpm utils <command> <subcommand> <arguments> <options>
```



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. `pnpm install`
2. Run the single code freeze command, `pnpm utils code-freeze verify-day`
3. Run tests `pnpm run --filter='./tools/monorepo-utils' test`
